jobs/build/ocp/Jenkinsfile

```diff
@@ -187,285 +242,346 @@ node(TARGET_NODE) {
-              )
-            }
+                // OIT Method
+                OIT_WORKING = "${pwd(tmp=true)}"
 
-            stage( "compare dist-git" ) {
-                sh "ose_images.sh --user ocp-build compare_nodocker --branch rhaos-${BUILD_VERSION}-rhel-7 --group base"
-            }
+                stage( "update dist-git" ) {
+                  sh "echo 'ose: ${env.OSE_DIR}' > ${env.WORKSPACE}/sources.yml"
+                  sh "echo 'jenkins: ${env.OPENSHIFT_JENKINS_DIR}' >> ${env.WORKSPACE}/sources.yml"
+                  sh "echo 'openshift-ansible: ${env.OPENSHIFT_ANSIBLE_DIR}' >> ${env.WORKSPACE}/sources.yml"
 
-            stage( "update dist-git" ) {
-                sh "ose_images.sh --user ocp-build update_docker --branch rhaos-${BUILD_VERSION}-rhel-7 --group base --force --release '${NEW_DOCKERFILE_RELEASE}' --version 'v${NEW_VERSION}'"
-            }
+                  oit """
+--working-dir ${OIT_WORKING} --group 'openshift-${BUILD_VERSION}' \\
+distgits:rebase --sources ${env.WORKSPACE}/sources.yml --version ${NEW_VERSION} \\
+--release ${NEW_DOCKERFILE_RELEASE} --no-push
+"""
+                }
 
-            stage( "build images" ) {
-                // TODO: Create a dynamic .repo file pointing to the exact puddle we built instead of "building" so that we can run X.Y builds in parallel
-                sh "ose_images.sh --user ocp-build build_container --branch rhaos-${BUILD_VERSION}-rhel-7 --group base --repo https://raw.githubusercontent.com/openshift/aos-cd-jobs/master/build-scripts/repo-conf/aos-unsigned-building.repo"
-            }
+                record_log = buildlib.parse_record_log( OIT_WORKING )
+                distgit_notify = buildlib.get_distgit_notify( record_log )
+
+                for(kv in buildlib.mapToList(distgit_notify)) {
+                    distgit = kv[0]
+                    val = kv[1]
+                    if(val.owners) {
+                        alias = val['source_alias']
+                        dockerfile_url = ""
+                        if ( alias ){
+                            github_url = env.GITHUB_URLS[alias]
+                            github_url = github_url.minus(".git")
+                            github_url = github_url.minus("git@")
+                            github_url = github_url.replaceFirst(":", "/")
+                            base_path = env.GITHUB_BASE_PATHS[alias]
+                            dockerfile_sub_path = val['dockerfile'].minus(base_path)
+                            dockerfile_url = "Source file: " + github_url + "/" + dockerfile_sub_path
+                        }
+                        mail(to: "${val.owners}",
+                                from: "aos-cd@redhat.com",
+                                subject: "${val.image} Dockerfile reconciliation.",
+                                body: """OIT has detected a change in the Dockerfile for ${val.image}
+                    ${dockerfile_url}
```

Comment:
""" will maintain this indentation whitepace in the email, which won't look right.